AudioManagerUtilities/src/CAmSocketHandler.cpp

```diff
@@ -659,7 +663,7 @@ am_Error_e CAmSocketHandler::removeTimer(const sh_timerHandle_t handle)
         if (it->handle == handle)
         {
             mListTimer.erase(it);
-            return removeFDPoll(handle);
+            return removeFDPoll(handle, true);
```

Comment:
If the caller wrote REMOVE or CLOSE explicitly in the second parameter then it's obvious right away without looking inside the function.